extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcTenantConfig.java

```diff
@@ -467,6 +477,37 @@ public void setPrincipalClaim(String principalClaim) {
         }
     }
 
+    @ConfigGroup
+    public static class Proxy {
+
+        /**
+         * The host (name or IP address) of the Proxy.<br/>
+         * Note: If OIDC adapter needs to use a Proxy to talk with OIDC server (Provider),
+         * then at least both "host" and "port" items must be configured to enable the usage of a Proxy.
+         */
+        @ConfigItem
+        public Optional<String> host = Optional.empty();
+
+        /**
+         * The port number of the Proxy.
+         */
+        @ConfigItem
+        public OptionalInt port = OptionalInt.of(80);
```

Comment:
Ah sorry, I didn't mean to remove the `defaultValue` from @ConfigItem. I was just suggesting to make it non-injection-environments friendly :)

Reply:
Np, but you lost me. Can you provide the concrete expectiation, please?

Sorry that I keep coming with non standard things in the code.
But eventually, after this small journey I will definitely want to contribute later to this wonderful project.

Reply:
No problem. I mean something like: 

```java 
@ConfigItem(defaultValue="80")
public int port = 80;
```

Reply:
So, would this work? 
```java
@ConfigItem
public int port = 80;
```
I'm a bit lost now ... Please, a last help here.

Reply:
Yes, it should work, but you need to add the `defaultValue` attribute in `@ConfigItem` AFAIK, as I showed in https://github.com/quarkusio/quarkus/pull/8120#discussion_r397866426

Reply:
```
@ConfigItem(defaultValue="80")
public int port;
```
? - we do it for booleans this way

Reply:
Sorry @gastaldi, @devisions, it should probably be
```
@ConfigItem(defaultValue="80")
public int port = 80;
```
as suggested by George if we want to make it non-injection env friendly (for TenantConfigResolver, etc)